include/options.hxx

```diff
@@ -104,14 +104,14 @@ using std::string;
 class Options {
 public:
   /// Constructor. This is called to create the root object
-  Options() : parent(nullptr) {}
-
+  Options();
+  
   /// Constructor used to create non-root objects
   ///
   /// @param[in] p         Parent object
   /// @param[in[ secname   Name of the section, including path from the root
-  Options(Options *p, string secname) : parent(p), sectionName(secname) {};
-
+  Options(Options *p, string s) : parent(p), sectionName(s) {};
```

Comment:
I think this renaming should go the other way: `parent` and `sectionName` instead of `p` and `s` :)

Reply:
Done